api/v1alpha1/performanceprofile_conversion.go

```diff
@@ -17,8 +17,9 @@ func (curr *PerformanceProfile) ConvertTo(dstRaw conversion.Hub) error {
 	dst.ObjectMeta = curr.ObjectMeta
 
 	// Spec
-	dst.Spec.CPU = new(v1.CPU)
 	if curr.Spec.CPU != nil {
+		dst.Spec.CPU = new(v1.CPU)
```

Comment:
I don't follow. We know `curr.Spec.CPU` is non-nil so... we initialize it again? (here and in a bunch of other places)

Reply:
we check the `curr` one, and if not nil we initialize the destination one.

Reply:
ah right, stupid me. Reading again.

Reply:
end of the week :)